client/packages/system/src/IndicatorsDemographics/api/operations.graphql

```diff
@@ -56,6 +56,7 @@ query demographicProjections(
   ) {
     ... on DemographicProjectionConnector {
       nodes {
+        __typename
```

Comment:
with this missing, the insert thought that it failed bc it is checking for a valid typename

Reply:
These need to be added on the insert mutation as well, getting the same error!